util/hot_backup.cc

```diff
@@ -0,0 +1,654 @@
+// -------------------------------------------------------------------
+//
+// hot_backup.cc
+//
+// Copyright (c) 2011-2016 Basho Technologies, Inc. All Rights Reserved.
```

Comment:
For new files, the copyright date should not be a range but rather it should be just the current year, since this file (and the corresponding .h file) were first created this year.